api/geth_backend.go

```diff
@@ -2496,7 +2466,9 @@ func (b *GethStatusBackend) getVerifiedWalletAccount(address, password string) (
 		return nil, wallettypes.ErrAccountDoesntExist
 	}
 
-	key, err := b.accountManager.VerifyAccountPassword(config.KeyStoreDir, address, password)
+	keystoreDirPath := filepath.Join(config.DataDir, config.KeyStoreDir)
```

Comment:
Can you please give a quick explanation, why?

I know we have confusion about absolute/relative paths: https://github.com/status-im/status-go/issues/5563

E.g. we definitely make `KeyStoreDir` absolute here: https://github.com/status-im/status-go/blob/e478786e7df17f74d80d966542d38d25e57565ef/api/geth_backend.go#L2186
While the `b.rootDataDir` is coming from client here: https://github.com/status-im/status-go/blob/c5293eb8468583cfae318d1062e8673a370b2ee0/mobile/status.go#L116